infrastructure/stage/config.ts

```diff
@@ -1,5 +1,5 @@
 import { getDefaultApiGatewayConfiguration } from '@orcabus/platform-cdk-constructs/api-gateway';
-import { StageName } from '@orcabus/platform-cdk-constructs/utils';
+import { StageName } from '@orcabus/platform-cdk-constructs/shared-config/accounts';
 import { VpcLookupOptions } from 'aws-cdk-lib/aws-ec2';
 import { SequenceRunManagerStackProps } from './stack';
 
```

Comment:
We have a few equivalent const in the `@orcabus/platform-cdk-constructs/shared-config/` if you wanted centralized 
```
  const vpcName = 'main-vpc';
  const vpcStackName = 'networking';
  const vpcProps: VpcLookupOptions = {
    vpcName: vpcName,
    tags: {
      Stack: vpcStackName,
    },
  };
  const computeSecurityGroupName = 'OrcaBusSharedComputeSecurityGroup';
  const eventBusName = 'OrcaBusMain';


```